app/uk/gov/hmrc/cgtpropertydisposals/connectors/UpscanConnector.scala

```diff
@@ -55,34 +40,57 @@ trait UpscanConnector {
 }
 
 @Singleton
-class UpscanConnectorImpl @Inject() (playHttpClient: PlayHttpClient, config: ServicesConfig)(
+class UpscanConnectorImpl @Inject() (
+  playHttpClient: PlayHttpClient,
+  config: ServicesConfig,
+  implicit val mat: ActorMaterializer
```

Comment:
not sure if this is going to affect anything, but what happens if you stick this in the same parameter group as the execution context?

Reply:
I did try that as well and I still get the same issue :upside_down_face:

Reply:
did you try injecting in `system: ActorSystem`?

Reply:
tried that as well